MyOwnORM/MyOwnORM/CustomDbSetReflection.cs

```diff
@@ -0,0 +1,693 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetReflection<T> where T : class
```

Comment:
don't use so many static, it is non-efficient at all, remove static from all methods and use this class as service

Reply:
Fixed